en/docs/samples/limiting-active-sessions-adaptive-auth.md

```diff
@@ -0,0 +1,82 @@
+# Configure Limiting Active User Sessions 
+
+This page guides you through setting up active user session limiting based on particular criteria for a sample application using sample authenticators. 
```

Comment:
Shall we change this ?
This page guides you through setting up an active user session for a sample application which is limited based on a particular criteria, using sample authenticators.

Reply:
That would give a different idea as we are not setting up an active user session for a sample app. 

Will change it to this to make it better:

This page guides you through setting up active user session limiting for a sample application based on a particular criteria, using sample authenticators.